hr_timesheet_sheet/tests/test_hr_timesheet_reviewer_no_email.py

```diff
@@ -0,0 +1,107 @@
+from .test_hr_timesheet_sheet import TestHrTimesheetSheet
+
+
+class TestHrTimesheetReviewerNoEmail(TestHrTimesheetSheet):
+    @classmethod
+    def setUpClass(cls):
+        super(TestHrTimesheetReviewerNoEmail, cls).setUpClass()
+        cls.timesheet_model_obj = cls.env["hr_timesheet.sheet"]
+        cls.mail_message_obj = cls.env["mail.message"]
+        cls.message_dict = {
+            "subject": "Test Message #1",
+            "body": "Test Message Content",
+            "model": "hr_timesheet.sheet",
+            "message_type": "notification",
+        }
+
+    def test_notify_compute_recipients(self):
```

Comment:
I think this test better split on the separate tests. Every tests will testing separate case.

Reply:
resolved